resources/tr_TR.yml

```diff
@@ -5,11 +5,13 @@ file-version: 1
 BANK_MAIN_PANEL_TITLE: "Klan bankası"
 BANK_DEPOSIT_TITLE: "Para yatır"
 BANK_WITHDRAW_TITLE: "Para çek"
+BANK_HISTORY_TITLE: "Klan geçmişi"
 BANK_DEPOSIT_INPUT: "Klan bankasına eklemek istediğiniz tutarı girin, geri dönmek için boş bir alan girin"
 BANK_WITHDRAW_INPUT: "Tutarı klan bankasına kaldırmak istediğiniz tutarı girin, geri dönmek için boş bir alan girin"
 BANK_MAIN_CONTENT: "§7Klan parası: §6{{money}}§7$"
-SUCCESS_BANK_DEPOSIT: " §a>> §2{{money}}$ başarıyla yatırdınız!"
-SUCCESS_BANK_WITHDRAW: " §a>> §2{{money}}$ başarıyla çektiniz!"
+BANK_HISTORY_CONTENT: " §a>> §2{{money}}$'ı başarıyla yatırdınız!"
```

Comment:
This should not be transcribed:
```
Here are all the operations carried out on the faction bank
```

Reply:
There were two new translation lines: 
* BUTTON_NEXT_HISTORY_BANK
* BUTTON_PREVIOUS_HISTORY_BANK